app/forms/PaymentsIntoOverseasPensionsFormProvider.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import play.api.data.Form
+
+import javax.inject.Singleton
+
+@Singleton
+class PaymentsIntoOverseasPensionsFormProvider {
+
+  def paymentsIntoOverseasPensionsForm(isAgent: Boolean): Form[Boolean] = YesNoForm.yesNoForm(
+    missingInputError = s"pensions.paymentsIntoOverseasPensions.statusPage.error.${if (isAgent) "agent" else "individual"}"
```

Comment:
Needs updating when `statusPage` is removed.

Reply:
Agree and will remove